esp/applications/eclwatch/testauth_authorization_map.yaml

```diff
@@ -0,0 +1,43 @@
+authNZ:
+   test:
+      name: test
+      type: test
+      LibName: libtestauthSecurity.so
+      authtype: AuthTypeMixed
+      # Define which authorization resources to be tested.
+      # As default, the authorization resources defined in
+      # ldap_authorization_map.yaml are tested.
+      useResourceMapsFrom: ldap
+      useraccess:
+      -  username: testadmin
+         #secretkey: testauthusercreds1     # optional
+         #vaultkey: testauthuservaultcreds1 # optional
+         defaults:
+            filescope: FULL
+            eclwuscope: FULL
+            resource: FULL
+      -  username: testuser
+         #secretkey: testauthusercreds2     # optional
+         #vaultkey: testauthuservaultcreds2 # optional
+         defaults:
+            filescope: NONE
```

Comment:
minor: The capitalisation isn't very consistent.  E.g., LibName is camel case with leading upper.  useResourceMapsFrom is camel with leading lower.  authtype, filescope etc. is all lower (rather than authType, fileScope)
Most entries in the values.yaml use camel case with leading lower.

Reply:
Some of them are based on the existing code and environment.xml. For example, system\security\shared\secloader.hpp(49): secMgrCfg->getProp("@LibName", libName);. We cannot change the environment.xml. Not sure we want to change the existing code to handle both cases. I will check them one by one to make it consistent as much as I can.

Reply:
Right.  If we change names that are currently a pass through to match old config then we need to create a mapping.  We've avoided that, but could do more of it if it's worth it.  If we do much more of it we can create a mapping file rather than mapping in c++.